file-server/app.py

```diff
@@ -0,0 +1,149 @@
+import os
+import json
+from dotenv import load_dotenv
+from flask import Flask, request, jsonify
+from flask_cors import CORS
+from imgget import search_images, download_images
+import voiceovergen as vo
+import moviegen as mg
+import llm_req as llm
+import userstuff as us
+
+load_dotenv()
+
+PORT = os.environ["FILE_SERVER_PORT"]
```

Comment:
why's it being called a file server and a file server port??

Reply:
Renaming to PORT